R/sotkanet_read_cache.R

```diff
@@ -0,0 +1,42 @@
+#' @title Read cache for sotkanet data.frame
+#'
+#' @description
+#' Helper function that reads the cache for saved sotkanet data.frame.
+#'
+#'
+#' @param cache a logical whether to do caching.
+#' @param cache_dir a path to cache directory.
+#' @param query_hash a character used to identify the data.frame.
+#'
+#' @references See citation("sotkanet")
```

Comment:
@references tag should be used for the following purpose: "Use @references point to published material about the package that users might find helpful." Source: https://mpn.metworx.com/packages/roxygen2/7.1.0/articles/rd.html

Putting package citation in References is actually also used in the eurostat package so we should look into whether this is sensible thing to do